src/content/docs/fr/tutorial/1-setup/4.mdx

```diff
@@ -0,0 +1,70 @@
+---
+type: tutorial
+title: Stockez votre dépôt en ligne
+description: |-
+  Tutoriel : Construisez votre premier blog Astro —
+  Créez un référentiel GitHub pour votre projet de tutoriel
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+<PreCheck>
+  - Mettez votre projet en ligne
+</PreCheck>
+
+Ce tutoriel utilisera GitHub pour stocker notre dépôt et se connecter à un hébergeur web. Vous êtes libre d'utiliser le fournisseur git en ligne de votre choix.
+
+:::note
+Si vous êtes déjà familier avec git et avez votre propre flux de travail, créez un nouveau référentiel GitHub pour votre projet en utilisant votre méthode préférée. Passez à la page suivante : ["Déployez votre site sur le Web"](/en/tutorial/1-setup/5/).
+:::
+
+## Créer un référentiel sur GitHub
+
+Bien qu'il existe plusieurs moyens de stocker votre code local sur GitHub, ce tutoriel vous guidera à travers une méthode qui ne nécessite pas l'utilisation de git en ligne de commande.
+
+1. Connectez-vous à GitHub.com dans un navigateur et cliquez sur le <kbd>+</kbd> en haut à droite de l'écran pour créer un nouveau référentiel.
+
+2. Choisissez un nom pour votre référentiel. Il n'a pas besoin d'avoir le même nom que le dossier de votre projet.
+
+3. Vous verrez différentes options, mais vous n'avez pas besoin de modifier les paramètres par défaut. Faites défiler vers le bas et cliquez sur le bouton <kbd>Créer le référentiel</kbd>.
+
+4. Vous verrez diverses étapes de configuration, mais vous n'aurez pas besoin de les utiliser. Notez l'URL de votre référentiel. Vous pouvez maintenant quitter cette page sans rien faire.
+
+## Validez votre code local sur GitHub
+
+Dans la dernière section, vous avez apporté une modification au contenu de votre page. Cela signifie que les fichiers de votre projet ont été modifiés, et VS Code devrait afficher un nombre en haut de l'icône du menu "Source". Cet onglet Source est l'endroit où vous irez régulièrement pour mettre à jour vos fichiers sur GitHub.
+
+1. Cliquez sur l'onglet Contrôle de source dans VS Code pour voir la liste des fichiers modifiés. Si vous voyez un message vous indiquant que vous devez installer `git`, suivez les instructions fournies, puis rechargez VS Code.
+
+2. Cliquez sur le menu des "3 points" <kbd>•••</kbd> au-dessus du message de validation et choisissez <kbd>Remote</kbd> > <kbd>Ajouter une télécommande</kbd>.
```

Comment:
Keep the English button name 
```suggestion
2. Cliquez sur le menu des "3 points" <kbd>•••</kbd> au-dessus du message de validation et choisissez <kbd>Remote</kbd> > <kbd>Add Remote</kbd>.
```

Or we can use the french VSCode translation (but it's not a really good translation to be honest):
```suggestion
2. Cliquez sur le menu des "3 points" <kbd>•••</kbd> au-dessus du message de validation et choisissez <kbd>À distance</kbd> > <kbd>Ajouter un dépôt distant</kbd>.
```